README.md

```diff
@@ -1,5 +1,24 @@
 amber-documentation
 ===================
 
-Amber documentation website
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/amber-smalltalk/amber-documentation?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+This is the repository of the [Amber documentation website](http://docs.amber-lang.net).
+
+The site is developed using [Jekyll](http://jekyllrb.com/).
+
+## How to add a page to this amber-documentation site?
+
+1. Meet requirements. You need Jekyll in your system: `gem install jekyll`.
+2. Clone this repo with `git clone git@github.com:amber-smalltalk/amber-documentation.git`.
```

Comment:
Maybe this should also mention that you need to fork on GH and push to your fork.
Otherwise it is not possible to open pull requests.